integration-tests/src/test/java/oracle/weblogic/kubernetes/ItFmwDomainInPVUsingWDT.java

```diff
@@ -0,0 +1,392 @@
+        DB_IMAGE_TO_USE_IN_SPEC, FMWINFRA_IMAGE_TO_USE_IN_SPEC);
+    assertDoesNotThrow(() -> setupDBandRCUschema(DB_IMAGE_TO_USE_IN_SPEC, FMWINFRA_IMAGE_TO_USE_IN_SPEC,
+        RCUSCHEMAPREFIX, dbNamespace, 0, dbUrl),
+        String.format("Failed to create RCU schema for prefix %s in the namespace %s with "
+            + "dbUrl %s", RCUSCHEMAPREFIX, dbNamespace, dbUrl));
+    logger.info("DB image: {0}, FMW image {1} used in the test",
+        DB_IMAGE_TO_USE_IN_SPEC, FMWINFRA_IMAGE_TO_USE_IN_SPEC);
+
+    // install operator and verify its running in ready state
+    installAndVerifyOperator(opNamespace, domainNamespace);
+
+    // create pull secrets for domainNamespace when running in non Kind Kubernetes cluster
+    // this secret is used only for non-kind cluster
+    createSecretForBaseImages(domainNamespace);
+  }
+
+  /**
+   * Create a basic FMW domain on PV using WDT.
+   * Verify Pod is ready and service exists for both admin server and managed servers.
+   * Verify EM console is accessible.
+   */
+  @Test
+  @DisplayName("Create a FMW domainon on PV using WDT")
+  public void testFmwDomainOnPVUsingWdt() {
+    final String pvName = domainUid + "-" + domainNamespace + "-pv";
+    final String pvcName = domainUid + "-" + domainNamespace + "-pvc";
+    final int t3ChannelPort = getNextFreePort(30000, 32767);
+
+    // create FMW domain credential secret
+    createSecretWithUsernamePassword(wlSecretName, domainNamespace,
+        ADMIN_USERNAME_DEFAULT, ADMIN_PASSWORD_DEFAULT);
+
+    // create RCU credential secret
+    createRcuSecretWithUsernamePassword(rcuSecretName, domainNamespace,
+        RCUSCHEMAUSERNAME, RCUSCHEMAPASSWORD, RCUSYSUSERNAME, RCUSYSPASSWORD);
+
+    // create persistent volume and persistent volume claim for domain
+    createPV(pvName, domainUid, this.getClass().getSimpleName());
+    createPVC(pvName, pvcName, domainUid, domainNamespace);
+
+    // create a model property file
+    File fmwModelPropFile = createWdtPropertyFile(t3ChannelPort);
+
+    // use shell script to download WDT and run the WDT createDomain script to create FMW domain
+    Path wdtScript = Paths.get(RESOURCE_DIR, "bash-scripts", wdtCreateDomainScript);
+    // WDT model file containing WebLogic domain configuration
+    Path wdtModelFile = Paths.get(MODEL_DIR, fmwModelFile);
+
+    // create configmap and domain in persistent volume
+    createDomainJobOnPv(wdtScript, wdtModelFile, fmwModelPropFile.toPath(), pvName, pvcName);
+
+    // create domain and verify
+    createDomainCrAndVerify(pvName, pvcName, t3ChannelPort);
+
+    // verify that all servers are ready and EM console is accessible
+    verifyDomainReady();
+  }
+
+  private void createDomainCrAndVerify(String pvName,
+                                       String pvcName,
+                                       int t3ChannelPort) {
+    // create a domain custom resource configuration object
+    logger.info("Creating domain custom resource");
```

Comment:
can we utilize oracle.weblogic.kubernetes.utils.FmwUtils createDomainCrAndVerify?

Reply:
I am afraid that I can't, Marina, createDomainCrAndVerifyn is for MII, also my test need the below env var that is not set in createDomainCrAndVerify

            .addEnvItem(new V1EnvVar()
                    .name("ALLOW_DYNAMIC_CLUSTER_IN_FMW")
                    .value("true"))

Reply:
Add an optional boolean parameter ( allowDynamicCluster) to the  FMWUtil.createDomainResource() to take care of extra env variable ALLOW_DYNAMIC_CLUSTER_IN_FMW

Reply:
The point is not only setup env var, ALLOW_DYNAMIC_CLUSTER_IN_FMW, FMWUtil.createDomainResource() creates a MII domain, this test need a domain on PV

Reply:
Add a new method createDomainResourceOnPv() in FmwUtils  to consolidate all the FMW domain utility.

Reply:
done